src/model.js

```diff
@@ -28,6 +28,18 @@ const xpost = async (req, res, next) => {
 		next(error);
 	}
 };
+
+const xget_meta = async (req, res, next) => {
+	try {
+		let query = {};
+		query['_box'] = req.box;
+		const records = await Data.find(query).exec();
```

Comment:
Instead of the getting all the records and then getting the dates, we should be executing 2 queries to get the first and last record alone.

Reply:
Do this we can get rid of the helper function as well.